payments-ui-core/src/main/java/com/stripe/android/ui/core/PaymentsTheme.kt

```diff
@@ -55,6 +53,61 @@ object PaymentsThemeConfig {
         val borderStrokeWidthSelected = 2.dp
     }
 
+    @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+    object Typography {
+        private val fontWeightBold: Int = FontWeight.Bold.weight
+        private val fontWeightMedium: Int = FontWeight.Medium.weight
+        private val fontWeightNormal: Int = FontWeight.Normal.weight
+        private val fontSizeMultiplier: Float = 1.0F
+        val fontFamily: Int = R.font.roboto
+
+        val h4 = TextStyle.Default.copy(
+            fontFamily = FontFamily(Font(fontFamily)),
+            fontSize = (20.0 * fontSizeMultiplier).sp,
+            fontWeight = FontWeight(fontWeightBold),
+        )
+
+        val h5 = TextStyle.Default.copy(
+            fontFamily = FontFamily(Font(fontFamily)),
+            fontSize = (16.0 * fontSizeMultiplier).sp,
+            fontWeight = FontWeight(fontWeightMedium),
+            letterSpacing = (-0.32).sp
```

Comment:
Are the letter spacing affected by accessibility font size? Just want to make sure that when the font size is set to largest that we aren't overlapping any letters.

Reply:
These were also used in the XML layouts. Double checked by changing both the hardcoded font sizes here in this PR and via the OS and didn't see any overlaps.